core/commands/partials/terraform_configuration_generator.rb

```diff
@@ -238,9 +238,11 @@ def retrieve_configuration_file_generator
     when 'aws'
       Result.ok(TerraformAwsGenerator.new(@configuration_id, @aws_config, @ui, @configuration_path, @ssh_keys))
     when 'gcp'
-      Result.ok(TerraformGcpGenerator.new(@configuration_id, @gcp_config, @ui, @configuration_path, @ssh_keys, @env.gcp_service))
+      Result.ok(TerraformGcpGenerator.new(@configuration_id, @gcp_config, @ui, @configuration_path,
```

Comment:
Please check the file with the Rubocop and fix all warnings.
`1 file inspected, 23 offenses detected`